### PR TITLE
Text drawable to handle text size 

### LINKED ISSUE #1006: Text drawable does not handle size
It seems impossible to change the size of the annotations with such graphics.

We should provide a specific method and handle it with the viewer

### PR DESCRIPTION
**Pull request by sofianehaddad**
It fixes #1006

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -85,6 +85,7 @@
  * #995 (Pip does not recognize conda install of openturns)
  * #1000 (SaltelliSensitivityAlgorithm with LowDiscrepancyExperiment produces wrong results)
  * #1005 (The stopping criteria of SobolSimulationAlgorithm is weird)
+ * #1006 (Text drawable does not handle size)
```
Comment (jschueller) [inline ChangeLog]: It's a feature so it will be in the next release.

Comment (sofianehaddad) [reply ChangeLog]: fine with me. I update the ChangeLog

Comment (regislebrun) [reply ChangeLog]: +1 good job, small enough to be in 1.13? Really in next release?

Comment (jschueller) [reply ChangeLog]: yup, can you update the changelog Sofiane ?

Comment (sofianehaddad) [reply ChangeLog]: Hmm didn't understand. Should I move it to another section?

Comment (jschueller) [reply ChangeLog]: Yes to the 1.14 section

Comment (sofianehaddad) [reply ChangeLog]: thus it will not be integrated to 1.13 as @regislebrun mentionned?

Comment (jschueller) [reply ChangeLog]: As it introduces new apis I prefer not to include it, sorry.

Comment (sofianehaddad) [reply ChangeLog]: ok, fine with me

Comment (sofianehaddad) [reply ChangeLog]: @jschueller done

**Pull request merged**